stories/datepicker/datepicker.stories.js

```diff
@@ -424,7 +429,12 @@ export const DefaultAndCompactSizes = () => `<div style="display:flex;justify-co
 
 DefaultAndCompactSizes.parameters = {
     docs: {
-        iframeHeight: 500
+        iframeHeight: 500,
+        description: {
+            story: 'The default date picker is represented on the left part of the container in the example below. The default version of date picker is automatically generated through `fd-docs-date-picker` class. In order to generate a compact version of the date picker, add `fd-input--compact` in the input class.'
+
```

Comment:
Here you need to modify a little bit the text. You can start with "The date picker provides responsive behavior that allows for simple operation on all devices. It is smaller in compact mode and provides a touch-friendly size in cozy mode." This I took from Responsiveness section in https://experience.sap.com/fiori-design-web/date-picker/
We don't mention the modifier classes for compact as they are many and applied on every element that is a building block of the Date Picker. 
OR :) 
you can add something like (just with the correct English :D )
"For compact mode add the appropriate modifier classes, such as `fd-input--compact`, `fd-button--compact`, `fd-calendar--compact`, etc to the building blocks of the component."